railties/lib/rails/generators/test_unit/system/templates/system_test.rb

```diff
@@ -0,0 +1,7 @@
+require "system_test_helper"
+
+class <%= class_name.pluralize %>Test < ActionSystemTestCase
+  # test 'the truth' do
+  #   assert true
+  # end
```

Comment:
Let's use a more realistic commented out example to give people a pointer, like:

```ruby
visit root_url
assert_text 'h1', 'Not there!'
```